src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Resolve paths with CloudSdk and Python defaults.
  */
-public enum PathResolver {
+public enum PathResolver implements IPathResolver {
```

Comment:
I think we should name it DefaultPathResolver and get rid of enum/singleton. CloudSdk will need to be modified to accept a PathResolver in the CloudSdk.Builder.